apisix/cli/ngx_tpl.lua

```diff
@@ -75,12 +75,27 @@ http {
                       .. [=[{*lua_cpath*};";
 
     {% if enabled_stream_plugins["prometheus"] then %}
+
+    init_by_lua_block {
+        require "resty.core"
+        apisix = require("apisix")
```

Comment:
I double-checked this line and it seems the `apisix` is not used?

Reply:
remove it, and this helped me to solve the problem of repeatedly requiring apisix, thanks.